quantum/main.c

```diff
@@ -63,6 +63,12 @@ int main(void) {
         // Run Quantum Painter animations
         void qp_internal_animation_tick(void);
         qp_internal_animation_tick();
+
```

Comment:
Introducing extra logic here would be nice to avoid and instead introduce something like a `qp_task()`